### DISCUSSION TITLE
Logins per server

### DISCUSSION THREAD

**Discussion by LOENS2** [gravitational/teleport#13403] [Q&A] [answered]
Is it possible to set up logins specific to one server? Many of my servers have dedicated service accounts, that are only used on one server. Having to reconfigure the user every time I set up a new server to add service accounts is a mess and i prefer not to do that.

**Comment by webvictim** [comment, marked as answer]
This isn't possible with the way Teleport works.

My recommendation would be to set up a common account across all servers that can be used to perform administrative tasks. Even if that login just has the ability to `sudo -u service_user -i` to perform management tasks it would make administration easier from the Teleport side.
